src/messages/label.ts

```diff
@@ -62,4 +62,8 @@ export const labels = {
   City: "City",
   Country: "country",
   listOfSupportedCamps: "List of supported camps",
+  otpLabel:
+    "Note : Verification code has been sent to your registered email address",
```

Comment:
All the mesages must come from the common message file.

Reply:
This message comes from the shared file.